service/models.py

```diff
@@ -15,26 +15,32 @@
 class DataValidationError(Exception):
     """ Used for an data validation errors when deserializing """
 
-    pass
 
-
-class YourResourceModel(db.Model):
+class Customer(db.Model):
     """
-    Class that represents a YourResourceModel
+    Class that represents a customers
     """
 
     app = None
 
     # Table Schema
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(63))
+    lastname = db.Column(db.String(63))
+    email = db.Column(db.String(120))
+    phone = db.Column(db.String(20))
+    address = db.Column(db.String(256))
```

Comment:
This should be split into 2 lines (line1 and line 2) like in the zenhub issue schema description